src/System.Private.Uri/src/System/UriHelper.cs

```diff
@@ -134,9 +134,6 @@ internal static class UriHelper
         internal static unsafe char[]? EscapeString(string input, int start, int end, char[]? dest, ref int destPos,
             bool isUriString, char force1, char force2, char rsvd)
         {
-            if (end - start >= Uri.c_MaxUriBufferSize)
```

Comment:
Are we still using this (`Uri.c_MaxUriBufferSize`) constant somewhere else? Otherwise, I would have expected this PR to remove this constant, and the associated `SR.net_uri_SizeLimit` error string.

Reply:
Yes, e.g.
https://github.com/dotnet/corefx/blob/9630ecefc14671d26b2426e9bc76b8178dd3dd75/src/System.Private.Uri/src/System/Uri.cs#L1923

Reply:
So, that is potentially another area of improvement for System.Uri. Perhaps that limit is no longer required?

Reply:
It might not be.  However, when I remove that and try larger things against APIs like Uri's ctor, I quickly bump up against several other ushort.MaxValue-like limits, which seem to be more "real".  So I constrained the fix to just these APIs that don't actually interact with Uri instances, just strings.